modules/core/src/test/java/org/apache/ignite/loadtests/communication/GridIoManagerBenchmark0.java

```diff
@@ -44,13 +44,15 @@
 import org.apache.ignite.spi.communication.tcp.TcpCommunicationSpi;
 import org.apache.ignite.testframework.GridTestUtils;
 import org.apache.ignite.testframework.junits.common.GridCommonAbstractTest;
+import org.junit.Ignore;
 import org.junit.Test;
 
 import static org.apache.ignite.internal.managers.communication.GridIoPolicy.PUBLIC_POOL;
 
 /**
  *
  */
+@Ignore("Benchmark")
```

Comment:
please file an additional ticket to deal with all benchmark/load tests, link to it here.

Reply:
@alamar, Same proposal here.
Let it be @Ignore("TBD XXX") where XXX is some hint, if possible.

Reply:
It's better to always have machine-readable JIRA URL as Ignore message. All the details may be also added in the form of Java comment.

Reply:
It's better, but this can be a mandatory requirement.

Reply:
Added jira url and comment.